ledger/src/shred.rs

```diff
@@ -21,10 +21,38 @@ use solana_sdk::{
     pubkey::Pubkey,
     signature::{Keypair, Signature, Signer},
 };
-use std::{mem::size_of, sync::Arc, time::Instant};
+use std::{mem::size_of, sync::Arc};
 
 use thiserror::Error;
 
+#[derive(Default, Clone)]
+pub struct ProcessShredsStats {
+    // Per-slot elapsed time
+    pub shredding_elapsed: u64,
+    pub receive_elapsed: u64,
+    pub serialize_elapsed: u64,
+    pub gen_data_elapsed: u64,
+    pub gen_coding_elapsed: u64,
+    pub sign_coding_elapsed: u64,
+    pub coding_send_elapsed: u64,
+    pub get_leader_schedule_elapsed: u64,
+}
+impl ProcessShredsStats {
+    pub fn update(&mut self, new_stats: &ProcessShredsStats) {
+        self.shredding_elapsed += new_stats.shredding_elapsed;
+        self.receive_elapsed += new_stats.receive_elapsed;
+        self.serialize_elapsed += new_stats.serialize_elapsed;
+        self.gen_data_elapsed += new_stats.gen_data_elapsed;
+        self.gen_coding_elapsed += new_stats.gen_coding_elapsed;
+        self.sign_coding_elapsed += new_stats.sign_coding_elapsed;
+        self.coding_send_elapsed += new_stats.gen_coding_elapsed;
```

Comment:
Looks like this is by mistake adding `gen_coding_elapsed` to `coding_send_elapsed`.
Putting a fix now.